.github/CODEOWNERS

```diff
@@ -0,0 +1,91 @@
+android/res/values-ar/ @organicmaps/translations-ar
+android/res/values-be/ @organicmaps/translations-be
+android/res/values-bg/ @organicmaps/translations-bg
+android/res/values-cs/ @organicmaps/translations-cs
+android/res/values-da/ @organicmaps/translations-da
+android/res/values-de/ @organicmaps/translations-de
+android/res/values-el/ @organicmaps/translations-el
+android/res/values-es/ @organicmaps/translations-es
+android/res/values-eu/ @organicmaps/translations-eu
+android/res/values-fa/ @organicmaps/translations-fa
+android/res/values-fi/ @organicmaps/translations-fi
+android/res/values-fr/ @organicmaps/translations-fr
+android/res/values-fr-rCA/ @organicmaps/translations-fr
+android/res/values-he/ @organicmaps/translations-he
+android/res/values-hu/ @organicmaps/translations-hu
+android/res/values-in/ @organicmaps/translations-id
```

Comment:
-in

Reply:
Indonesian has `in` code on Android and `id` code on iOS:

- https://github.com/championswimmer/android-locales
- https://gist.github.com/jacobbubu/1836273